images/ose-installer-kube-apiserver-artifacts.yaml

```diff
@@ -0,0 +1,46 @@
+base_only: true
+content:
+  source:
+    dockerfile: openshift-hack/images/installer-kube-apiserver-artifacts/Dockerfile.rhel
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/kubernetes.git
+      web: https://github.com/openshift/kubernetes
+    modifications:
+    - action: replace
+      match: "COPY . ."
+      replacement: "COPY . .\nRUN rm -rf unpacked_remote_sources"  # FIXME: Workaround for CLOUDBLD-8974
+    ci_alignment:
+      streams_prs:
+        enabled: false
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: ose-installer-kube-apiserver-artifacts-container
+enabled_repos:
+- rhel-9-baseos-rpms
+- rhel-9-appstream-rpms
+for_payload: false
+for_release: false
+from:
+  builder:
+  - stream: rhel-9-golang
+  - stream: rhel-9-golang
+  - stream: rhel-9-golang
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base
```

Comment:
This would need to be `openshift-enterprise-base-rhel9`

Reply:
Fixed.